utils/kvrocks2redis/sync.cc

```diff
@@ -180,8 +180,12 @@ Status Sync::incrementBatchLoop() {
         if (bulk_data_str != "ping") {
           auto bat = rocksdb::WriteBatch(bulk_data_str);
           int count = static_cast<int>(bat.Count());
-          parser_->ParseWriteBatch(bulk_data_str);
-          auto s = updateNextSeq(next_seq_ + count);
+          auto s = parser_->ParseWriteBatch(bulk_data_str);
+          if (!s.IsOK()) {
+            return s.Prefixed(fmt::format("failed to parse write batch '{}'", Util::StringToHex(bulk_data_str)));
```

Comment:
Does it help to print out the write batch? This is a long string, and I personally don't think it's very helpful for troubleshooting.

Reply:
@caipengbo I've just recalled that we use the same approach in replication [(here)](https://github.com/apache/incubator-kvrocks/blob/unstable/src/cluster/replication.cc#L552) and thought that perhaps it was intended to help with debugging.

Reply:
Get it, I did encounter this error in the replication, because there is a background error, replica can not be written in. I found that the string is very long, and it is not useful for troubleshoting problem, maybe you said, is used for debugging.

Reply:
Yes, it can help us inspect the batch if it goes wrong, so it's fine.